checkov/terraform/checks/resource/aws/IAMManagedAdminPolicy.py

```diff
@@ -0,0 +1,48 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+ADMIN_POLICY_NAME = "AdministratorAccess"
+ADMIN_POLICY_ARN = f"arn:aws:iam::aws:policy/{ADMIN_POLICY_NAME}"
+
+
+class IAMManagedAdminPolicy(BaseResourceCheck):
+    def __init__(self):
+        # This is the full description of your check
+        description = "Disallow IAM roles, users, and groups from using the AWS AdministratorAccess policy"
+
+        # This is the Unique ID for your check
+        id = "CKV_AWS_273"
```

Comment:
```suggestion
        id = "CKV_AWS_275"
```
sorry `CKV_AWS_273` is already in use